### ISSUE TITLE
Checkbox filled with black in Edge browser

### ISSUE THREAD

**Issue by deap82** [material-components/material-components-web#1212]
### What MDC-Web Version are you using?
0.19.0

### What browser(s) is this bug affecting?
Edge  40.15063.0.0

### What OS are you using?
Windows 10

### What are the steps to reproduce the bug?
Check this demo in Edge:
https://material-components-web.appspot.com/checkbox.html

### What is the expected behavior?
The checkbox should be outlined.

### What is the actual behavior?
The checkbox is filled with black, especially notable in the non dark theme examples.

**Comment by trungvt-pnl** [comment]
I have faced up to this problem. In my case, there is no problem to remake the style of checkbox little bits: fix the checkbox background in `white` even unchecked or checked state and set the checkmark color to `black`. One more thing, I use a css trick for applying to only `Edge`.
This works for me:
```
@supports (-ms-ime-align:auto) { // only MS Edge +12!! Yesss!!
    .mdc-checkbox__background {
        background-color: white !important; // set the filled background of checkbox
        .mdc-checkbox__checkmark__path {
            stroke: #59513a !important; // set the stroke color of checkmark
        }
    }
```
Hope this help. Please let me know if you have any ideas.

**Comment by trungvt** [comment]
I have faced up to this problem. In my case, there is no problem to remake the style of checkbox little bits: fix the checkbox background in white even unchecked or checked state and set the checkmark color to light orange (it's up to you). One more thing, I use a css trick for applying to only Edge.
This works for me:
```
@supports (-ms-ime-align:auto) { // only MS Edge +12!! Yesss!!
    .mdc-checkbox__background {
        background-color: white !important; // set the filled background of checkbox
        .mdc-checkbox__checkmark__path {
            stroke: #59513a !important; // set the stroke color of checkmark
        }
    }
}
```
Hope this help. Please let me know if you have any ideas.

**Comment by kfranqueiro** [comment]
Thanks for the report.

I believe this is related to Edge's buggy behavior with regard to CSS variables used within pseudo class selectors, along with #1102 and #664.

**Comment by kfranqueiro** [comment]
This has been fixed via #1364 which was merged manually.
